src/components/meeting.jsx

```diff
@@ -25,6 +26,19 @@ export default class Meeting extends Component {
 		this.setState({ viewport: viewport });
 	}
 
+	getMeetingLink(meeting) {
+		return (
+			<a href={ window.location.pathname + '?meeting=' + meeting.slug } onClick={event => this.setMeeting(event, meeting.slug)}>{meeting.name}</a>
+		);
+	}
+
+	setMeeting(event, slug) {
+		event.preventDefault();
+		console.log(this.props.state.input);
+		this.props.state.input.meeting = slug;
+		this.props.setAppState('input', this.props.state.input);
+	}
+
```

Comment:
I'm going to move this block to a new component since it's duplicated in the other file. i see in your history you named it `MeetingLink` -- I'm going to be cheeky and name it `Link` for now but if we have lots of other links we can change it